routes/index.js

```diff
@@ -6,9 +6,9 @@ import cheerio from 'cheerio';
 const router = express.Router();
 
 function urlIsValid(urlString) {
-  const parsed = url.parse(urlString);
-  return parsed.hostname === 'rightmove.co.uk' ||
-    parsed.hostname === 'www.rightmove.co.uk';
+  const hostname = url.parse(urlString).hostname;
+
+  return hostname === 'rightmove.co.uk' || hostname === 'www.rightmove.co.uk';
```

Comment:
That would allow any subdomain (or super-domain, e.g. `rightmove.co.uk.lolhacking.com` through). I think we do want to be stricter here.